src/project/internal/projectmigrator.cpp

```diff
@@ -134,6 +138,55 @@ void ProjectMigrator::resetStyleSettings(mu::engraving::MasterScore* score)
     score->resetStyleValue(mu::engraving::Sid::measureSpacing);
 }
 
+void ProjectMigrator::map440HeaderFooterStyles(MasterScore* score)
+{
+    IF_ASSERT_FAILED(score) {
+        return;
+    }
+
+    // Copyright and page numbers used header/footer styling before 4.4 - after 4.4 these have their own styles. To ensure nothing
+    // changes visually when loading a pre-4.4 score for the first time, we must search the header/footer strings for copyright/page
+    // number macros and set the "defaults" for copyright/page number styles based on where the macros were inserted...
+    const auto doMap = [](MasterScore* score, const TextStyleType type, const std::vector<Sid> headerFooterStringSids) {
```

Comment:
Nitpicks: `score` could be captured instead of passed by argument, and the vector could be `const std::vector<Sid>&`